README.md

```diff
@@ -145,15 +146,15 @@ Explaining what each part does:
   * `noargs` doesn't throw an error when retrieving blank arguments from the game (needed for blackops4.exe)
   * `waitforexit` leave bnetlauncher open and waiting until the game existing (needed for destiny 2 to show you as playing)
   * `nolaunch` don't directly launch the game but just open the client and try to find the game for an additional 60s. This can be 
-    used launch a game and give time to select a region or other unsuported options.
+    used launch a game and give time to select a region or other unsupported options.
   * `notask` doesn't start the client trough a scheduled task, this will make the steam overlay also apply to the battle.net client
   * `noadmin` tries to apply compatibility flags to the game to avoid calling the UAC, this is an untested hack that can break the game
   **do not use unless you know what you're doing**.
 
 ## Special Thanks
 
 * internet coder Maruf for ghost tray icon fix code
-* github Ethan-BB for the new parameters to launch games on battle.net
```

Comment:
github here is to identify the platform of the username, it should not be removed.